cmd/cql/internal/help.go

```diff
@@ -0,0 +1,37 @@
+package internal
+
+import (
+	"fmt"
+	"runtime"
+)
+
+const name = "cql"
+
+var (
+	// Version of command, set by main func of version
+	Version = "unknown"
+)
+
+var CmdVersion = &Command{
+	UsageLine:   "cql version",
+	Description: "Show cql build version infomation",
+}
+
+func init() {
+	CmdVersion.Run = runVersion
+}
+
+func PrintVersion(printLog bool) string {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported function PrintVersion should have comment or be unexported